src/components/Nav/Navcategory.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import Navlinkcategory from './Navlinkcategory';
+import './Navcategory.scss';
+
+const Navcategory = () => {
+  return (
+    <div className="navCategory">
+      <div className="sideCategory">
+        <span className="category">
+          <i class="fa-solid fa-bars"></i> 카테고리
+        </span>
+      </div>
+      <div className="itemCategory">
+        {NAVLIST.map((item) => (
+          <Navlinkcategory key={item.id} name={item.name} />
+        ))}
+      </div>
+      <div className="luciferParcel">
+        <span className="lucifer">샛별 택배 </span>
+        <span className="parcel"> 배송안내</span>
+      </div>
+    </div>
+  );
+};
+export default Navcategory;
+
+const NAVLIST = [
+  { id: 'new', name: '신상품' },
+  { id: 'best', name: '베스트' },
+  { id: 'Sale', name: '알뜰쇼핑' },
+  { id: 'Benefits', name: '특가/헤택' },
```

Comment:
props로 id의 내용이 UI로 보여져야 한다고 가정했을때
이처럼 구현하는 것이 이해가 되지만, id에 해당하는 값이 단순 key prop 에만 쓰이고 있으니
문자가 아닌 숫자로 id를 주는 것이 더 좋을 것 같습니다.

```suggestion
  { id: 1, name: '신상품' },
  { id: 2, name: '베스트' },
  { id: 3, name: '알뜰쇼핑' },
  { id: 4, name: '특가/헤택' },
```

Reply:
id 숫자로 반영하겠습니다!!